app/controllers/posts_controller.rb

```diff
@@ -88,7 +92,10 @@ def new
   end
 
   def create
-    import_thread and return if params[:button_import].present?
+    if params[:button_import].present?
```

Comment:
Why did this change?

Reply:
`import_thread` stopped returning a truthy value all the time, so in some circumstances `import_thread and return` was failing to return.

I could just make this line: https://github.com/Marri/glowfic/pull/665/files#diff-ab4e9273f7df75aa5004a42f757d6f51R313 into `return true if performed?`?

Reply:
(Solved by converting `require_import_permission` to a `before_action`.)